### PR TITLE
User/vijay1

### PR DESCRIPTION
**Pull request by VijayaraghavanSundaravaradan**
PR review for https://polestarjira.atlassian.net/browse/DT-5813

### CODE REVIEW

**Comment by AntonJansson675** [comment]
Strange, that you can't add me as a reviewer? @VijayaraghavanSundaravaradan

**Comment by VijayaraghavanSundaravaradan** [comment]
@AntonJansson675 Now I can see you here.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
 name: Build and attach
-
-on: [push]
+#temp modification to avoid too many workflow triggers
```
Comment (AntonJansson675) [inline .github/workflows/build.yml:2]: Should it still be temp? :)

File: .github/workflows/buildandupload.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        run: |
+          gh api \
+          --method POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          /repos/${{github.repository}}/releases \
+          -f tag_name='v1.0.${{github.RUN_ID}}' \
+          -f target_commitish='${{github.sha}}' \
+          -f name='v1.0.${{github.RUN_ID}}' \
+          -f body='Release Description here' \
```
Comment (AntonJansson675) [inline .github/workflows/buildandupload.yml:65]: out of curiosity, what is the difference between capital F and f in this context?

Comment (VijayaraghavanSundaravaradan) [reply .github/workflows/buildandupload.yml:65]: Its from the syntax from GH Documentation here: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release

We can try it small and capital letters and see if there is a difference.

File: .github/workflows/build.yml (hunk 1 above)
Comment (VijayaraghavanSundaravaradan) [reply .github/workflows/build.yml:2]: Can remove the comment. But still have to think about the trigger to be set.

File: .github/workflows/buildandupload.yml (hunk 2 above)
Comment (AntonJansson675) [reply .github/workflows/buildandupload.yml:65]: No need to go that far for it. :)

**Review by AntonJansson675 — APPROVED** [review]

**Pull request merged**